HW1/P2/P2b.py

```diff
@@ -1,3 +1,28 @@
 from P2 import *
 
 # Your code here
+import pyspark
+import numpy as np
+import random
+import matplotlib.pyplot as plt
+
+sc = pyspark.SparkContext(appName='myAppName')
+
+i = np.arange(1,2001)
+j = np.arange(1,2001)
+
+random.shuffle(i)
+random.shuffle(j)
+
```

Comment:
Why don't you do the shuffle inside Spark? For example, you can call a partitionBy with a shuffle on the index on the rdd intensity. Then you won't have any issue if the data set is too large. And also that way the work is even more balanced.